cfme/tests/candu/test_utilization_metrics.py

```diff
@@ -302,3 +316,90 @@ def test_raw_metric_host_disk(metrics_collection, appliance, provider):
         if record.disk_usage_rate_average is not None:
             assert record.disk_usage_rate_average > 0, 'Zero Host Disk IO'
             break
+
+
+def query_metric_rollup_table(appliance, provider, metric, azone_name):
+    metrics_tbl = appliance.db.client['metric_rollups']
```

Comment:
This needs a docblock explaining in plain English what the intent of the query is, and what's being returned.

Reply:
Done